Code/GraphMol/new_canon.h

```diff
@@ -96,13 +97,20 @@ struct RDKIT_GRAPHMOL_EXPORT canon_atom {
         isRingStereoAtom(false),
         nbrIds(NULL),
         p_symbol(NULL){};
+
+  ~canon_atom() {
+    if (nbrIds != nullptr) {
```

Comment:
don't need the test for nullptr
Might think about making `nbrIds` an `std::unique_ptr`

Reply:
Right.

But I think a `std::vector<int>`would be more appropriate than a `std::unique_ptr`.